src/pygeoprocessing/routing/routing.pyx

```diff
@@ -3056,12 +3057,17 @@ def extract_strahler_streams_d8(
             the above arguments. Contains the fields "order" and "parent" as
             described above.
         min_flow_accum_threshold (int): minimum number of upstream pixels
-            required to create a stream, the final value may be adjusted based
-            on significant differences in 1st and 2nd order streams.
+            required to create a stream. If `autotune_flow_accumulation`

```

Comment:
Double-backticks?

Reply:
okay just added in cb2c8e25b7d104d0a221b16bebea9dc54a96506b (used the web editor actually)